src/Sylius/Bundle/ApiBundle/Serializer/ProductImageNormalizer.php

```diff
@@ -80,7 +80,7 @@ private function resolvePath($data): array
             /** @var string $filter */
             $filter = $request->query->get('filter');
 
-            $data['path'] = $this->cacheManager->getBrowserPath(parse_url($data['path'], \PHP_URL_PATH), $filter);
+            $data['path'] = $this->cacheManager->getBrowserPath((string) parse_url($data['path'], \PHP_URL_PATH), $filter);
```

Comment:
According to [`parse_url`](https://www.php.net/manual/en/function.parse-url.php) doc, it will return `string|null|false` with `\PHP_URL_PATH`, so type casting result will introduce possible error and we should instead:

1.  Ensure, that `$data['path`]` is not null and is a not empty string.
2. Handle possible `false` on seriously malformed URLs.